tests/integration/passthrough-test.js

```diff
@@ -202,3 +202,25 @@ test('it can passthrough other-origin hosts', function(assert) {
   });
 });
 
+
+test('passthrough is not overridable', function(assert) {
+  assert.expect(1);
+  var done1 = assert.async();
+  var server = this.server;
+
+  server.loadConfig(function() {
+    this.urlPrefix = '/vet/123';
```

Comment:
I think wanted to test if it works with `fullPath` but actually forgot to put `/vet/123/pets` in ajax url